WordPress/Classes/ViewRelated/Blog/KeyringAccountController.swift

```diff
@@ -0,0 +1,53 @@
+import Foundation
+import WordPressShared
+
+
+/// KeyringAccount is used to normalize the list of avaiable accounts while
+/// preserving the owning keyring connection.
+///
+struct KeyringAccount {
+    var name: String // The account name
+    var externalID: String? // The actual externalID value that should be passed when creating/updating a publicize connection.
+    var externalIDForConnection: String // The effective external ID that should be used for comparing a keyring account with a PublicizeConnection.
+    var keyringConnection: KeyringConnection
+}
+
+@objc class KeyringAccountController: NSObject {
```

Comment:
Would you mind explaining why this class had to exist? I'm also unsure of `*Controller` suffix here. That usually denotes a `ViewController` subclass but this class is not.

Reply:
This is a helper class, it encapsulates logic for KeringAccount. I've seen that you don't have a ViewModel or a Presenter so i don't think that UI(include view controller) needs to know about this logic. Another reason is that i want to rehuse code and avoid duplicating that. Let me change that to KeyringAccountHelper, what do you think about it?